<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FCurrencyEdit</class>
 <widget class="QDialog" name="FCurrencyEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Icon</string>
       </property>
       <property name="buddy">
        <cstring>pwIcons</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name</string>
       </property>
       <property name="buddy">
        <cstring>leName</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="ui::icons::WIconSelect" name="pwIcons" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="leName"/>
       </item>
       <item>
        <widget class="QToolButton" name="tbClearName">
         <property name="toolTip">
          <string>Clear name</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../../res/hacc.qrc">
           <normaloff>:/main/icons/edit-clear-locationbar-rtl.png</normaloff>:/main/icons/edit-clear-locationbar-rtl.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Reduction</string>
     </property>
     <property name="buddy">
      <cstring>leReduction</cstring>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="leReduction"/>
     </item>
     <item>
      <widget class="QToolButton" name="tbClearReduction">
       <property name="toolTip">
        <string>Clear name</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../res/hacc.qrc">
         <normaloff>:/main/icons/edit-clear-locationbar-rtl.png</normaloff>:/main/icons/edit-clear-locationbar-rtl.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbReductionBeforeNumber">
     <property name="text">
      <string>Reduction before number</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>14</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ui::icons::WIconSelect</class>
   <extends>QWidget</extends>
   <header>wiconselect.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../res/hacc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FCurrencyEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>22</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FCurrencyEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbClearReduction</sender>
   <signal>clicked()</signal>
   <receiver>leReduction</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbClearName</sender>
   <signal>clicked()</signal>
   <receiver>leName</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
